cmd/eib/main.go

```diff
@@ -1,7 +1,85 @@
 package main
 
-import "fmt"
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+const (
+	argConfigFile = "config"
+	argVerbose    = "verbose"
+)
+
+func init() {
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:    true,
+		QuoteEmptyFields: true,
+	})
+	log.SetOutput(os.Stdout)
+}
+
+func parseFlags(programName string, args []string) (imageConfig *config.ImageConfig, output string, err error) {
```

Comment:
This function does a whole lot more than just parsing of flags. Adding more flags to it and using them for business logic will make this grow exponentially.

One approach is to leave the parsing in the main function and to create separate functions covering their usages e.g. loading the config.

Reply:
The reason I pulled them out of main was so I could more easily test them. I do see that point that its also loading the image config. Lemme think about that, but part of me just wants to leave it as is and refactor once we have more flags.